tests/chainer_tests/test_link.py

```diff
@@ -1142,6 +1142,7 @@ def _assert_arrays_equal(self, array, expected_array):
     def test_cpu_to_intel64(self):
         link = self.link
         link.to_intel64()
+        self.assertEqual(link._device_id, None)
```

Comment:
Use `assert`, and also `is` instead of `==` to compare with `None`.
(Ditto for others)

Reply:
Thanks, fixed.